.prettierrc

```diff
@@ -0,0 +1,6 @@
+{
+  "singleQuote": true,
+  "trailingComma": "all",
+  "semi": true,
```

Comment:
```suggestion
```

Do u want this option to use the default ?

https://prettier.io/docs/en/options.html#semicolons